File: june14/july3ev_Rec2/reverseArray.cpp
#include<iostream>
using namespace std;

void reverseRecursively(int arr[],int l,int r)
{
    if(l>=r)
        return;
    int temp=arr[l];
    arr[l]=arr[r];
    arr[r]=temp;
    reverseRecursively(arr,l+1,r-1);
}
int main(int argc,char** argv)
{
    int arr[7]={2,5,6,7,8,9,4};
    reverseRecursively(arr,0,6);
    for(int i=0;i<7;i++)
        cout << arr[i] << " ";
}

File: feb2/feb10_Array/reversearray.java
package feb10;

public class reversearray {

	public static void main(String[] args) {
		int[] arr={ 10,20,30,40,50};
		
		int n=arr.length;
		for(int i=0,j=n-1;i<j;i++,j--){
				int temp=arr[i];
				arr[i]=arr[j];
				arr[j]=temp;
		}
		for(int i=0;i<n;i++)
			System.out.println(arr[i]);

	}

}
